# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../alsa-utils/fix_weak_alsaconf_script.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
bash dislikes that '' -ne 0
Alejandro

--- ./alsaconf/alsaconf.in.orig	2005-02-19 23:03:30.000000000 -0300
+++ ./alsaconf/alsaconf.in	2005-02-19 23:04:55.000000000 -0300
@@ -54,9 +54,9 @@
   distribution="debian"
 elif [ -f /etc/mandrake-release ]; then
   distribution="mandrake"
-elif [ -f /etc/redhat-release -a `grep -c "Red Hat" /etc/redhat-release 2>/dev/null` -ne 0 ]; then
+elif [ -f /etc/redhat-release -a "`grep -c 'Red Hat' /etc/redhat-release 2>/dev/null`" != 0 ]; then
   distribution="redhat"
-elif [ -f /etc/fedora-release -a `grep -c "Fedora" /etc/fedora-release 2>/dev/null` -ne 0 ]; then
+elif [ -f /etc/fedora-release -a "`grep -c 'Fedora' /etc/fedora-release 2>/dev/null`" != 0 ]; then
   distribution="fedora"
 else
   distribution="unknown"
